### PR TITLE
EUI-5205 Linked hearings journey display of linked page

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5205
https://tools.hmcts.net/jira/browse/EUI-5616

### Change description ###
Linked hearings-Is the hearing linked in request hearing journey
Request hearing- Linked question on CYAP

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/hearings/containers/request-hearing/hearing-link/hearing-link.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { provideMockStore } from '@ngrx/store/testing';
+import { of } from 'rxjs';
+import { initialState } from 'src/hearings/hearing.test.data';
+import { ACTION } from 'src/hearings/models/hearings.enum';
+import { HearingsService } from 'src/hearings/services/hearings.service';
```
Comment (rayliangatos) [inline src/hearings/containers/request-hearing/hearing-link/hearing-link.component.spec.ts:10]: src

File: src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    this.hearingStore.pipe(select(fromHearingStore.getHearingsFeatureState)).subscribe(
+      state => {
+        this.caseId = state.hearingList.hearingListMainModel ? state.hearingList.hearingListMainModel.caseRef : '';
+      }
+    );
+  }
+
+  public ngOnInit(): void {
+    this.hearingLinkForm = this.formBuilder.group({
+      hearingLink: ['', Validators.required]
```
Comment (rayliangatos) [inline src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts:39]: by default it should also prepopulate what is in the store hearingDetails.hearingIsLinkedFlag

**Review by rayliangatos — CHANGES_REQUESTED** [review]

File: src/hearings/containers/request-hearing/hearing-link/hearing-link.component.spec.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/hearings/containers/request-hearing/hearing-link/hearing-link.component.spec.ts]: I have done the changes.

File: src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts (hunk 2 above)
Comment (johnbenjamin-hmcts) [reply src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts]: Thank you @rayliangatos I have done the changes.

**Review by rayliangatos — APPROVED** [review]
thanks for addressing the missing requirement, looks good to me.

**Review by SivaprakasamSivaprakasam — APPROVED** [review]

**Review by Kartheek-S — APPROVED** [review]

**Pull request merged**